server/tests/api/test_organizations.py

```
@@ -15,3 +17,29 @@ def test_invite_with_invalid_role_id(client):
 
     invite = OrganizationDataHandler.get_invites(orgId=1)
     assert len(invite) == 0
+
+
+def test_invite_does_not_wait_for_email(client, monkeypatch):
+    email_started = Event()
+    release_email = Event()
+
+    def blocking_send(_email):
```

Ruff (0.16.2)

[warning] Missing return type annotation for private function `blocking_send`

Add return type annotation: `bool`

(ANN202)